### DISCUSSION TITLE
me salta fatal error: (nombre del archivo) no such file or directory

### DISCUSSION THREAD

**Discussion by diegoperez2004** [INGCOM-UNRN-P1/cursada-2023#53] [Preguntas y respuestas] [answered]
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141845473/e7cfc61e-eeca-4a06-8f2b-a34841cf1d97)
estaba haciendo un ejercicio del tp1 y me salta este error al compilar. hoy cuando estaba resolviendo unos ejercicios todo andaba perfectamente hasta que me salto este error. fui a un ejercicio que habia resuelto anteriormente (el de la imagen) y tambien me aparecio este error.

**Comment by jingusci** [comment]
Buenas! no se si lo pudiste resolver, pero parece como que el compilador no encuentra el archivo "ejercicio4.c". 
Es probable que la consola no esté en la carpeta donde están los ejercicios.

**Comment by martinvilu** [comment, marked as answer]
Falta información, estás usando vscode o la consola suelta.

Cual es la 'ruta' hacia los archivos.

**Reply by diegoperez2004** [reply]
Estoy usando vscode.
Lo raro es que no me daba ningun problema al compilar y derepente dejo de funcionar. Tambien probe compilar por el git bash y me dice lo mismo. No se si toque algo yo o que

**Reply by martinvilu** [reply]
Subí el código al repositorio y comparti la direccion asi lo revisamos

**Reply by martinvilu** [reply]
¿Esta cerrado esto?
